ci/unit_tests/workspace_addjson/main_test.py

```diff
@@ -0,0 +1,58 @@
+
+import os, unittest, sys, codecs, json
+
+import workspace_addjson
+
+class TestMain(unittest.TestCase):
+
+    def test_addingJsonDictionary(self):
+        ''' Tests for including additional json dictionary to arbitrary location in dialog nodes '''
+
+        cmd = 'cp ./ci/unit_tests/workspace_addjson/main_data/workspace.json ./ci/unit_tests/workspace_addjson/main_data/workspace_forAddJsonTest.json'
```

Comment:
Path to folder with data could be defined as class member and reused everywhere, it would increase readability and portability of code.

Reply:
fixed. also for workspaceFile and copy command